TestON/drivers/common/cli/onosclidriver.py

```diff
@@ -1256,7 +1256,7 @@ def removeIntent( self, intentId ):
             cli output otherwise
         """
         try:
-            cmdStr = "remove-intent " + str( intentId )
+            cmdStr = "remove-intent org.onosproject.cli " + str( intentId )
```

Comment:
It may be helpful later on when we have other applications that install intents to be able to specify the application. Maybe we could add this as a keyword parameter with the default being cli?